### PR TITLE
[Core] Adding Redhat9 support

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Added redhat9 support

## How was this patch tested?

- canary build + created a jenkins cluster

### CODE REVIEW

**Comment by agl29** [comment]
@wing2fly @ranade1 please review this PR.

**Review by JohanAhlen — APPROVED** [review]
Nice!

File: tools/cloudera/build_hue_common.sh [hunk 1]
```diff
@@ -301,3 +310,33 @@ function ubuntu20_install() {
     export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
   fi
 }
+
+function redhat9_install() {
+  if [[ $FORCEINSTALL -eq 1 ]]; then
+    # pre-req install
+    sudo -- sh -c 'yum install -y \
+      java-11-openjdk \
```
[the remark this answers is not in the document]
Comment (Harshg999) [reply tools/cloudera/build_hue_common.sh:318]: Maybe for other OS packages?

[the remark this answers is not in the document]
Comment (wing2fly) [reply tools/cloudera/build_hue_common.sh:318]: In Makefile, we have maven install

File: tools/relocatable.sh [hunk 2]
```diff
@@ -83,15 +83,15 @@ fi
 
 export PATH=$(dirname $ENV_PYTHON):$PATH
 
-PYVER=$($ENV_PYTHON -V 2>&1|head -c 10)
+PYVER=$($ENV_PYTHON -V 2>&1 | awk '{print $2}' | cut -d '.' -f 1,2)
 # Step 1. Fix virtualenv
-if [[ $PYVER == "Python 3.8" ]]; then
+if [ "$(echo "$PYVER >= 3.8" | bc -l)" -eq 1 ]; then
```
Comment (wing2fly) [inline tools/relocatable.sh:88]: This works on all OSes?

**Review by wing2fly — APPROVED** [review]

File: tools/relocatable.sh (hunk 2 above)
Comment (agl29) [reply tools/relocatable.sh:88]: Yes, checked on ubuntu20 and centos7

**Pull request merged**